img/pictograms/hourglass.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@CrisTofani we need you help: could it be possible to include this svg in our icon-pack?

Reply:
@fabriziofff Since the png is used, is this svg needed?

Reply:
Due to legacy configuration of `react-native-typescript-transformer` the svg file could not be imported (https://github.com/kristerkari/react-native-svg-transformer#installation-and-configuration) and was replaced by a png.
removed in https://github.com/pagopa/io-app/pull/1994/commits/e5af8d55f5de3fabef6c969c6b2e2a7a781be0bc